service/src/test/java/uk/nhs/adaptors/gp2gp/uat/EhrExtractUATTest.java

```diff
@@ -158,37 +105,35 @@ public void tearDown() {
     @ParameterizedTest
     @MethodSource("testValueFilePaths")
     public void When_MappingValidJsonRequestBody_Expect_ValidXmlOutput(String inputJson, String expectedOutputXml) throws IOException {
-        String expectedJsonToXmlContent = ResourceTestFileUtils.getFileContent(OUTPUT_PATH + FILES_PREFIX + expectedOutputXml);
-        String inputJsonFileContent = ResourceTestFileUtils.getFileContent(INPUT_PATH + FILES_PREFIX + inputJson);
-        Bundle bundle = new FhirParseService().parseResource(inputJsonFileContent, Bundle.class);
+        final String expectedJsonToXmlContent = ResourceTestFileUtils.getFileContent(OUTPUT_PATH + FILES_PREFIX + expectedOutputXml);
+        final String inputJsonFileContent = ResourceTestFileUtils.getFileContent(INPUT_PATH + FILES_PREFIX + inputJson);
+        final Bundle bundle = new FhirParseService().parseResource(inputJsonFileContent, Bundle.class);
+
         messageContext.initialize(bundle);
 
-        EhrExtractTemplateParameters ehrExtractTemplateParameters = ehrExtractMapper.mapBundleToEhrFhirExtractParams(
-            getGpcStructuredTaskDefinition,
-            bundle);
-        String ehrExtractContent = ehrExtractMapper.mapEhrExtractToXml(ehrExtractTemplateParameters);
-        var hl7TranslatedResponse = outputMessageWrapperMapper.map(
-            getGpcStructuredTaskDefinition,
-            ehrExtractContent);
+        final EhrExtractTemplateParameters ehrExtractTemplateParameters =
+            ehrExtractMapper.mapBundleToEhrFhirExtractParams(getGpcStructuredTaskDefinition, bundle);
+
+        final String ehrExtractContent = ehrExtractMapper.mapEhrExtractToXml(ehrExtractTemplateParameters);
+
+        final String hl7TranslatedResponse = outputMessageWrapperMapper.map(getGpcStructuredTaskDefinition, ehrExtractContent);
 
-        assertThat(hl7TranslatedResponse).isEqualTo(expectedJsonToXmlContent);
+        assertThat(hl7TranslatedResponse).isEqualToIgnoringWhitespace(expectedJsonToXmlContent);
```

Comment:
This will need to be changed back to `isEqualTo`

Reply:
changed back